build.gradle.kts

```diff
@@ -63,6 +73,12 @@ tasks.test {
     }
 }
 
+val pitest = tasks.withType<info.solidsoft.gradle.pitest.PitestTask> {
```

Comment:
Tests failed:
```
10:24:00 AM PIT >> INFO : MINION : java.lang.AssertionError: [thruput per ms] 
10:24:00 AM PIT >> INFO : MINION : Expecting:
10:24:00 AM PIT >> INFO : MINION :  <1978.9049f>
10:24:00 AM PIT >> INFO : MINION : to be greater than:
10:24:00 AM PIT >> INFO : MINION :  <2500.0f> 
```
It seems the mutations are slowing down the benchmark. I guess we can drop the threshold?

Reply:
This plugin is spamming the stdout. A `./gradlew build` should be basically empty unless there's an actionable for the dev, like a red test or a warning.

- [ ] reduce/remove stdout spam

Reply:
We could exclude test from mutation. It's not checking the logic, but performance. Mutations are more useful for checking uncovered logical cases.